src/main/java/edu/ucsb/cs156/happiercows/controllers/CommonsController.java

```diff
@@ -221,6 +223,9 @@ public ResponseEntity<String> joinCommon(
 
         userCommonsRepository.save(uc);
 
+        joinedCommons.setNumUsers(joinedCommons.getNumUsers() + 1);
```

Comment:
This approach is not acceptable.   It adds redundant state to the Commons, and violated database normalization principles.  There is already a method on the commonsRepository for getting the number of  users.  That is how this must be implemented.   Other teams were forced to change their code after implementing this approach.